CMakeLists.txt

```diff
@@ -223,7 +223,7 @@ function(add_mhp_ctest_impl)
   if(DEFINED AMC_TIMEOUT)
     set_tests_properties(${AMC_TEST_NAME} PROPERTIES TIMEOUT ${AMC_TIMEOUT})
   else()
-    set_tests_properties(${AMC_TEST_NAME} PROPERTIES TIMEOUT 60)
+    set_tests_properties(${AMC_TEST_NAME} PROPERTIES TIMEOUT 600)
```

Comment:
you've changed default timeout for all tests
please change timeout just for long tests (TIMEOUT option for add_mhp_test

Reply:
as I told you in a private message this is just for testing purposes, I'll change it